docker/root/etc/s6/gitea/setup

```diff
@@ -30,6 +30,9 @@ if [ ! -f ${GITEA_CUSTOM}/conf/app.ini ]; then
     if [ -n "$SECRET_KEY" ] && [ -z "$INSTALL_LOCK" ]; then
         INSTALL_LOCK=true
     fi
+    if  [  "$DB_PASSWD_FILE" ]; then
+        DB_PASSWD=`cat $DB_PASSWD_FILE`;
```

Comment:
```suggestion
        DB_PASSWD="$(cat $DB_PASSWD_FILE)";
```

backtick syntax is legacy.

Reply:
Before making more changes, I would suggest to use existing mechanisms. See my comments above.

Reply:
Yeah I guess there are better solutions than to introduce `_FILE` variables for every variable.